aodntools/timeseries_products/timeseries_tools.py

```diff
@@ -0,0 +1,187 @@
+## Utility functions for timeseries products
```

Comment:
Please remove this file from this PR. It isn't being used yet and is just adding more duplicated code. We can do the refactoring in a separate PR.

Reply:
removed